crowbar_framework/lib/crowbar/backup/restore.rb

```diff
@@ -66,10 +66,21 @@ def restore_chef
       def restore_crowbar
         Crowbar::Backup::Base.restore_files.each do |source, destination|
           # keep the permissions of the files that are already in place
+          src_path = @data.join("crowbar", source)
+          dest_path = Pathname.new(destination)
+          src_string = ""
+          # If source and destination are both directories we just need to 
+          # copy the contents of source, not the directory itself.
+          if dest_path.directory? && src_path.directory?
```

Comment:
Style/ConditionalAssignment: Use the return of the conditional for variable assignment and comparison.